backend/migrations/dataall_migrations/herder.py

```diff
@@ -0,0 +1,79 @@
+import importlib
+import importlib.machinery
+import inspect
+import os
+from pathlib import Path
+from migrations.dataall_migrations.base_migration import BaseDataAllMigration
+
+import logging
+
+logger = logging.getLogger()
+logger.setLevel(os.environ.get('LOG_LEVEL', 'INFO'))
+
+
+class Herder:
+    def __init__(self):
+        self.migration_path = {}
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        self.folder_path = os.path.join(dir_path, 'versions')
+        self.initial_key = '0'
+        self.last_key = None
+
+        logger.info('Loading migrations...')
+        logger.info(f'Folder path: {self.folder_path}')
+
+        for py_file in Path(self.folder_path).glob('*.py'):
```

Comment:
will this get the __init__ file?

Reply:
@dlpzx It will, but there is nothing there. It is only for module init purposes.